Local failing unit test: views/sign_up - render - positions password help tooltip on the second password field

> AssertionError: expected '' to equal 'auto'

I expect this is caused by a similar problem to https://github.com/mozilla/fxa-content-server/issues/6900

--- a/app/tests/spec/views/sign_up.js
+++ b/app/tests/spec/views/sign_up.js
@@ -175,7 +175,9 @@ define(function (require, exports, module) {
             view.highlightSignupPasswordHelper({
               target: '#vpassword'
             });
-            assert.equal(view.$('.input-help-balloon').css('top'), 'auto');
+            /* The behaviour of firefox in new versions has changed,
+            so for versions <= 64 it is auto and for versions >= 65 it is '' (empty string) */
+            assert.include(['', 'auto'], view.$('.input-help-balloon').css('top'));
           });
       });
 

